jobs/service-fabrik-backup-manager/templates/bin/job_properties.sh.erb

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+#
+# Service-Fabrik-Backup-Manager properties
+#
+
+# Directory to store the configuration files
+export CONF_DIR=$JOB_DIR/config
+
+# Node environment
+export NODE_ENV=production
+
+# configuration file location
+export SF_EVENTMESH_SETTINGS_PATH=$CONF_DIR/settings.yml
+export SETTINGS_PATH=$CONF_DIR/settings.yml
```

Comment:
Why is the same file path exported in 2 different vars?

Reply:
Not required. Removed!